packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -1,98 +1,129 @@
-- [`L1ERC721Bridge.k.sol`](./proofs/L1ERC721Bridge.k.sol): Symbolic property tests for [`L1ERC721Bridge`](../../src/L1/L1ERC721Bridge.sol)
-- [`L1StandardBridge.k.sol`](./proofs/L1StandardBridge.k.sol): Symbolic property tests for [`L1StandardBridge`](../../src/L1/L1StandardBridge.sol)
-- [`OptimismPortal.k.sol`](./proofs/OptimismPortal.k.sol): Symbolic property tests for [`OptimismPortal`](../../src/L1/OptimismPortal.sol)
-- [`interfaces`](./proofs/interfaces): Files with the signature of the functions involved in the verification effort
-- [`utils`](./proofs/utils): Proof dependencies, including the summary contracts
+1. `cd` to the root of this repo.
+2. Install Foundry by running `pnpm install:foundry`. This installs `foundryup`, the foundry toolchain installer, then installs the required foundry version.
+3. Install Kontrol by running `pnpm install:kontrol`. This installs `kup`, the package manager for RV tools, then installs the required kontrol version.
+4. Install Docker.
 
-### [`scripts`](./scripts) folder
+## Usage
 
-- [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh): Executes [`KontrolDeployment.sol`](./KontrolDeployment.sol), curates the result and writes the summary deployment contract
-- [`run-kontrol.sh`](./scrpts/run-kontrol.sh): Proof execution script
-- [`json`](./scripts/json): Data cleaning scripts for the output of [`KontrolDeployment.sol`](./KontrolDeployment.sol)
+Verifying proofs has two steps: build, and execute.
 
-## Verification execution
+### Build Deployment Summary
 
-The verification execution consists of two steps, although the first step may be omitted to use the committed version. There's one script to run per step. These scripts should be run from the [`contracts-bedrock`](../../) directory.
+First, generate a deployment summary contract from the deploy script in [`KontrolDeployment.sol`](./KontrolDeployment.sol) by running the following command:
 
-1. Generate a deployment summary contract from [`KontrolDeployment.sol`](./KontrolDeployment.sol)
 ```bash
-  ./test/kontrol/scripts/make-summary-deployment.sh
+./test/kontrol/scripts/make-summary-deployment.sh
 ```
-This step is optional. The default summary can be found [here](./proofs/utils/DeploymentSummary.sol), which is the summarization of the [`KontrolDeployment.sol`](./KontrolDeployment.sol) script.
 
-2. Execute the tests in [`OptimismPortal.k.sol`](./proofs/OptimismPortal.k.sol)
-```bash
-  ./test/kontrol/scripts/run-kontrol.sh $option
+[`KontrolDeployment.sol`](./deployment/KontrolDeployment.sol) contains the minimal deployment sequence required by the proofs.
+The [`make-summary-deployment.sh`](./scripts/make-summary-deployment.sh) script will generate a summary contract is used to load the post-`setUp` state directly into Kontrol.
+
+This step is optional if an up-to-date summary contract already exists, which will be the case until the `KontrolDeployment` contract changes.
```

Comment:
If any of the tested contracts changes, it should also be the case that the JSON state diff (and thence the deployment contract) changes, since the etched code will be different

Reply:
Clarified in 1d24faf45